CODEOWNERS

```diff
@@ -114,10 +114,19 @@ ttnn/ @eyonland @arakhmati @cfjchu @xanderchin @TT-BrianLiu @ayerofieiev-tt @dma
 ttnn/**/kernels/ # Removes the owners above from owning kernels unless specified afterwards
 ttnn/setup.py @tt-rkim
 ttnn/CMakeLists.txt @tt-rkim @ayerofieiev-tt
+ttnn/cpp/ttnn/operations/ccl/ @SeanNijjar @TT-BrianLiu
+ttnn/cpp/ttnn/operations/reduce_scatter/ @SeanNijjar @TT-BrianLiu
+ttnn/cpp/ttnn/operations/pool/ @mywoodstock @nsmithtt
+ttnn/cpp/ttnn/operations/upsample/ @mywoodstock @nsmithtt
+ttnn/cpp/ttnn/operations/conv2d/ @mywoodstock @nsmithtt
```

Comment:
please replace @nsmithtt with @pavlejosipovic

Reply:
I had left @nsmithtt as just a backup in case both you and @pavlepopovic are on vacation.

Reply:
Removing @nsmithtt